app/models/comment.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  belongs_to :post
+end
```

Comment:
- [ ] Since comments belongs to User we need to add `belongs_to :user` here.

- [ ] It' makes sense to add validation at least for the comment content.